application/jira/v0/config/tasks.json

```diff
@@ -0,0 +1,808 @@
+          "instillUIOrder": 7,
+          "instillFormat": "string",
+          "title": "Status",
+          "type": "string"
+        }
+      },
+      "required": [
+        "id",
+        "key",
+        "self",
+        "fields"
+      ],
+      "title": "Issue",
+      "type": "object"
+    },
+    "sprint": {
+      "properties": {
+        "id": {
+          "title": "ID",
+          "description": "The ID of the sprint",
+          "type": "integer",
+          "instillUIOrder": 0,
+          "instillFormat": "integer"
+        },
+        "self": {
+          "title": "Self",
+          "description": "The URL of the sprint",
+          "type": "string",
+          "instillUIOrder": 1,
+          "instillFormat": "string"
+        },
+        "state": {
+          "title": "State",
+          "description": "The state of the sprint, can be: `active`, `closed`, `future`",
+          "type": "string",
+          "instillUIOrder": 2,
+          "instillFormat": "string"
+        },
+        "name": {
+          "title": "Name",
+          "description": "The name of the sprint",
+          "type": "string",
+          "instillUIOrder": 3,
+          "instillFormat": "string"
+        },
+        "start-date": {
+          "title": "Start Date",
+          "description": "The start date of the sprint. In the RFC3339 format, e.g. 2018-03-05T00:00:00Z",
+          "type": "string",
+          "instillUIOrder": 4,
+          "instillFormat": "string"
+        },
+        "end-date": {
+          "title": "End Date",
+          "description": "The end date of the sprint. In the RFC3339 format, e.g. 2018-03-05T00:00:00Z",
+          "type": "string",
+          "instillUIOrder": 5,
+          "instillFormat": "string"
+        },
+        "complete-date": {
+          "title": "Complete Date",
+          "description": "The complete date of the sprint. In the RFC3339 format, e.g. 2018-03-05T00:00:00Z",
+          "type": "string",
+          "instillUIOrder": 6,
+          "instillFormat": "string"
+        },
+        "origin-board-id": {
+          "title": "Origin Board ID",
+          "description": "The ID of the origin board",
+          "type": "integer",
+          "instillUIOrder": 7,
+          "instillFormat": "integer"
+        },
+        "goal": {
+          "title": "Goal",
+          "description": "The Goal of the sprint",
+          "type": "string",
+          "instillUIOrder": 8,
+          "instillFormat": "string"
+        }
+      },
+      "required": [
+        "id",
+        "self"
+      ],
+      "title": "Sprint",
+      "type": "object"
+    }
+  },
+  "TASK_LIST_BOARDS": {
+    "instillShortDescription": "List all boards in Jira",
+    "input": {
+      "description": "List all boards in Jira",
+      "instillUIOrder": 0,
+      "instillEditOnNodeFields": [],
+      "properties": {
+        "project-key-or-id": {
+          "default": "",
+          "title": "Project Key or ID",
```

Comment:
Important.
I forgot if I mentioned here.
But, please make sure how the users get project key or id.